docs/5-Quick_Reference.adoc

```diff
@@ -1,176 +1,176 @@
-|*{toSortedSet}*     |Converts the collection to a *MutableSortedSet* implementation and sorts it using the natural order of the elements.
-|*{toSortedSetBy}*   |Converts the collection to a *MutableSortedSet* implementation and sorts it based on the natural order of the attributes returned by the *Function*.
-|*{zip}*             |Takes a second *RichIterable* and pairs up all the elements; if one of the two **RichIterable**s is longer, its remaining elements are ignored.
-|*{zipWithIndex}*    |Zips the collection with the Integer indexes 0 to n-1.
-|===
-+++*+++ on *MutableCollection* +
-+++**+++ on *MutableList*
-
-[cols=",",options="header",]
-[%autowidth]
-|===
-|*Interfaces*    |*Description*
-|*{Bag}*         |Analogous to JCF _Map_ <K, Integer>.
-|*{Multimap}*    |Analogous to JCF _Map_ <K, Iterable<V>>.
-|*{MutableList}* |Analogous to JCF _List_ .
-|*{MutableMap}*  |Analogous to JCF _Map_ .
-|*{MutableSet}*  |Analogous to JCF _Set_ .
-|*{Pair}*        |Container that holds two related objects.
-|===
-
-[cols=",",options="header",]
-[%autowidth]
-|===
-|*Blocks*               |*Description*
-|*{Function}*             |Transforms one type (T) to another (V).
-|*{Function0}*            |A zero-argument lambda (Java 8) that returns some type; a factory.
-|*{Function2}*            |Used by *injectInto* methods; takes the accumulator argument as the first argument, and the current item of the collection as the second argument.
-|*{ObjectIntProcedure}*   |Takes an int as a second argument; this is usually the index of the current element of a collection; doesn't return anything.
-|*{Predicate}*            |Takes an object of some type (T) and returns a boolean.
-|*{Predicate2}*           |Primarily used in methods like *selectWith*, *detectWith*, *rejectWith*. The first argument is the element of the collection being iterated over, and the second argument is a parameter passed into the *Predicate2* from the calling method.
-|*{Procedure}*            |Takes an object of some type (T) and doesn't return anything.
-|*{Procedure2}*           |Takes two objects of types (T1, T2) and doesn't return anything.
-|===
-
-[cols=",",options="header",]
-[%autowidth]
-|===
-|*Static utility Classes* |*Interoperate with*
-|*{ArrayIterate}*         |Arrays
-|*{Iterate}*              |All iterables
-|*{LazyIterate}*          |All iterables for lazy evaluation
-|*{ListIterate}*          |Lists
-|*{MapIterate}*           |Maps
-|*{ParallelArrayIterate}* |Arrays for parallel iteration
-|*{ParallelIterate}*      |Iterables for parallel iteration
-|*{ParallelMapIterate}*   |Maps for parallel iteration
-|*{StringIterate}*        |strings
-|===
-
-
-== Eclipse Collections mindmap
-
-[link=EclipseCollectionsMindMap.png]
-image::EclipseCollectionsMindMap.png[Mindmap,800,945]
-
-[cols="3,^1",]
-[%autowidth]
-|===
-|xref:4-Testing_Utilities.adoc[previous: Unit testing]  |xref:0-RefGuide.adoc[top] 
-|===
+////
+   Copyright (c) 2022 The Bank of New York Mellon.
```

Comment:
The original copyright was correct as some of the content was from the original markdown version of the guide.